Wave/api/serializers.py

```diff
@@ -0,0 +1,39 @@
+from rest_framework import serializers
+from core.models import Post, Comment
+from users.models import User
+
+class UserSerializer(serializers.ModelSerializer):
+
+    class Meta:
+        model = User
+        fields = ('username', 'first_name', 'last_name', 'bio', 'github')
```

Comment:
remember to include email in this - I think?

Reply:
100%!